packages/cli/src/util.ts

```diff
@@ -231,6 +233,9 @@ export async function awsGetClusterRegionWithCmdlineFallback(): Promise<string>
           "command line parameter."
       );
     }
+
+    // Re-throw all other errors, mainly programming errors.
+    throw e;
```

Comment:
This one probably fixes https://github.com/opstrace/opstrace/issues/740 @sreis :shrug: